src/atoms/label/label.register.ts

```diff
@@ -0,0 +1,3 @@
+import { Label } from './label.js';
+
+window.customElements.define('skf-label', Label);
```

Comment:
I asked around a little, and I think it might be good to check if a component is defined before defining it, so as to avoid any double registering which can cause errors to be thrown. There might be another long term-solution that involves polyfilling a proposed feature on separate registries for each shadow dom... but that feels like overkill right now!

```suggestion
if(!window.customElements.get('skf-label')) {
  window.customElements.define('skf-label', Label);
}
```